docs/api/python/lambda_function.rst

```diff
@@ -0,0 +1,25 @@
+Lambda Function
```

Comment:
Is it worth including a setup section (e.g. what roles / permissions to configure with AWS), similar to SageMaker (https://www.run.house/docs/api/python/cluster#sagemaker-hardware-setup) or On-demand (https://www.run.house/docs/api/python/cluster#ondemandcluster-hardware-setup)?